bot/exts/utils/reminders.py

```diff
@@ -37,11 +38,43 @@
 LOCK_NAMESPACE = "reminder"
 WHITELISTED_CHANNELS = Guild.reminder_whitelist
 MAXIMUM_REMINDERS = 5
+REMINDER_EDIT_CONFIRMATION_TIMEOUT = 60
 
 Mentionable = discord.Member | discord.Role
 ReminderMention = UnambiguousUser | discord.Role
 
 
+class ModifyReminderConfirmationView(discord.ui.View):
+    """A view to confirm modifying someone else's reminder by admins."""
+
+    def __init__(self, author: discord.Member | discord.User):
+        super().__init__(timeout=REMINDER_EDIT_CONFIRMATION_TIMEOUT)
+        self.author = author
+        self.result: bool | None = None
+
+    async def interaction_check(self, interaction: Interaction) -> bool:
+        """Only allow interactions from the command invoker."""
```

Comment:
The first sentence of a function's docstring should complete this sentence: `This function will...`

Reply:
Could you please elaborate on this? What could the docstring be changed to instead? Would something like "Prevent other users from using the view buttons." be better?

Reply:
Actually, it could be left as is, so don't mind me.